components/Blocks/PhotoHero.js

```diff
@@ -1,13 +1,15 @@
-import React, {PropTypes} from 'react';
+import React, { PropTypes } from 'react';
 
 import styles from './PhotoHero.css';
 
 export default class PhotoHero {
   render() {
-    const {src, children} = this.props;
+    const { src, children } = this.props;
     return (
-      <div className={styles.root}
-           style={{backgroundImage: `url(${src})`}}>
+      <div
+        className={styles.root}
+        style={{ backgroundImage: `url(${src})` }}
```

Comment:
The next rules we were breaking were [jsx-indent-props](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-indent-props.md) and [jsx-closing-bracket-location](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-closing-bracket-location.md).

This makes prop indentation easy:
- If they fit on one line, keep them on one line:
```jsx
<Component foo="bar"/>
```

- If they don't fit, indent them with 2 spaces and close the component on the same column as it opens:
```jsx
<Component
  foo="bar"
>
  Children!
</Component>
```